.docker/scripts/create-test-data.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env sh
```

Comment:
these data provisioning should be done as custom step definitions class in BDD rather than this script.

It should have steps like:
```
Given users:
|name|email|role|
|test_admin||test_admin@example.com|admin|
|test_user||test_user@example.com|user|
....
```

Then, copy this file to all extensions and use it in all repos.

But for now it will suffice